modules/core/src/main/java/org/apache/ignite/internal/util/IgniteUtils.java

```diff
@@ -7786,6 +7802,71 @@ public static boolean p2pLoader(ClassLoader ldr) {
         return ldr instanceof GridDeploymentInfo;
     }
 
+    /**
+     * Returns Deployment class loader id if method was invoked in the job context or {@code null}
+     * if nothing context found or Deployment is switched off.
```

Comment:
typo: nothing -> no context was found or similar.

Reply:
Changed the message.